Atomic/storage.py

```diff
@@ -63,6 +63,9 @@ def cli(subparser):
                          help=_("remove all unused block devices from storage pool"))
     modifyp.add_argument('--driver', dest="driver", default=None, help='The storage backend driver', choices=['devicemapper', 'overlay', 'overlay2'])
     modifyp.add_argument('--vgroup', dest="vgroup", default=None, help='The storage volume group')
+    modifyp.add_argument("--graph", dest="graph",
+                        default=default_docker_lib(),
+                        help=_("Root of the Docker runtime (Default: %s)" % default_docker_lib()))
```

Comment:
Is this related to the test work? Looks like the opt was documented but never implemented?